domain/src/main/java/life/qbic/apps/datamanager/services/UserRegistrationService.java

```diff
@@ -0,0 +1,41 @@
+package life.qbic.apps.datamanager.services;
+
+import java.util.Arrays;
+import life.qbic.domain.events.DomainEventProducer;
+import life.qbic.domain.events.DomainEventSubscriber;
+import life.qbic.domain.usermanagement.DomainRegistry;
+import life.qbic.domain.usermanagement.registration.UserRegistered;
+
+/**
+ * <b><class short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
+ */
+public class UserRegistrationService {
+
+  public UserRegistrationService() {
+    super();
+  }
+
+  public void registerNewUser(final String fullName, final String email, final char[] password) throws ServiceException {
```

Comment:
```suggestion
  public void registerUser(final String fullName, final String email, final char[] password) throws ServiceException {
```
I don't think you can register a user that already exists. So the method name can be shortened.

Reply:
Addressed now, thank you!